### PR TITLE
269 Change get all field names function

### PR DESCRIPTION
**Pull request by rawad1989**

### CODE REVIEW

**Comment by rawad1989** [comment]
[(0.25) Admin - LogEntryAdmin - 'Options' object has no attribute 'get_all_field_names' [Cycle12]](https://trello.com/c/R1jUFENz/269-0-25-admin-logentryadmin-options-object-has-no-attribute-get-all-field-names-cycle12)

**Comment by nasief** [comment]
[(0.25) Emails - More emails getting flagged as Junk [Cycle13]](https://trello.com/c/kvP8kUxr/298-0-25-emails-more-emails-getting-flagged-as-junk-cycle13)

File: accountsplus/admin.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ class LogEntryAdmin(django.contrib.admin.ModelAdmin):
     list_display = ('action_time', 'user', 'content_type', 'object_link', 'action', 'change_message', )
 
     def get_readonly_fields(self, request, obj=None):
-        return django.contrib.admin.models.LogEntry._meta.get_all_field_names()
+        return [ f.name for f in django.contrib.admin.models.LogEntry._meta.get_fields() ]
```
Comment (awartani) [inline accountsplus/admin.py]: @rawad1989 no spaces after `[` and before `]`

File: accountsplus/templates/accounts/locked_out.html [hunk 2]
```diff
@@ -2,10 +2,6 @@
 {% load static from staticfiles %}
 {% load i18n %}
 
-{% block extrastyle %}
-    <link rel="stylesheet" href="{% static 'css/admin.css' %}">
```
Comment (awartani) [inline accountsplus/templates/accounts/locked_out.html:6]: @rawad1989 @nasief why did you remove this?

**Review by awartani — APPROVED** [review]

File: accountsplus/templates/accounts/locked_out.html (hunk 2 above)
Comment (nasief) [reply accountsplus/templates/accounts/locked_out.html:6]: @awartani I don't recall why I did it. This was maybe 1000 BC 😛

Comment (rawad1989) [inline accountsplus/templates/accounts/locked_out.html]: @nasief Do you remember why ?

Comment (rawad1989) [reply accountsplus/templates/accounts/locked_out.html:6]: @awartani Me too I'm trying to remember why

File: accountsplus/admin.py (hunk 1 above)
Comment (rawad1989) [reply accountsplus/admin.py]: @awartani Modified
